forum/qa-plugin/report-reason/script.js

```diff
@@ -0,0 +1,95 @@
+$(document).ready(function()
+{
+    $(".qa-form-light-button-flag").attr("type", "button");
+    
+    $(".qa-form-light-button-flag").click( function()
+    {
+        var postId = $(this).data("postid");
```

Comment:
Nie ma potrzeby każdorazowo pisać `$(this)` - wystarczy zrobić to raz i przypisać do zmiennej.